public/styles/component.scss

```diff
@@ -120,20 +120,25 @@ kbn-dis doc-table .kbnDocViewer__warning {
     flex-grow: 0;
 }
 
-.osButtonsStyle {
-    display: grid;
-    grid-template-columns: repeat(5, 1fr);
-    grid-gap: 10px;
+.message {
+    margin-top: 10px;
+    display: flex;
+    flex-direction: row;
 }
 
-.osButtonsStyleMac {
+.euiButtonGroup__buttons {
     display: grid;
-    grid-template-columns: repeat(3, 1fr);
+    grid-template-columns: repeat(5, 1fr);
     grid-gap: 10px;
+    padding: 0 3px;
+    box-shadow: none;
 }
```

Comment:
**question**: is this a class of the @elastic/eui? If it does, we should ensure only to affect the UI of the Wazuh plugin. This could be modifying the styles of plugins of the platform.

Reply:
Fixed in this [PR](https://github.com/wazuh/wazuh-kibana-app/pull/4837)